chapter08/mdw/Ex3.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+sub new_dh {
+        my ( $directory_name ) = @_;
+        opendir my $new_dh, $directory_name or die "Could not open $directory_name: $!";
+        return $new_dh
+}
```

Comment:
Functions are useful for separating out common behaviour, although when it's just one command you might not need it.